src/System.IO.Pipelines/src/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard1.1;
+      netstandard1.3;
```

Comment:
I don't think this is right. By doing this we are effectively dropping support of netstandard1.1 which is a breaking change, specially when we are still building a netstandard1.1 ref.

Reply:
> By doing this we are effectively dropping support of netstandard1.1 which is a breaking change

This assembly is new this release.